README.md

```diff
@@ -4,7 +4,7 @@ This repo contains sample use case projects using the [OpenTok Insights API].
 
 ## Welcome to OpenTok
 
-If you're new to OpenTok, you can [sign up for a Tokbox account](https://tokbox.com/account/user/signup?utm_source=DEV_REL&utm_medium=github&utm_campaign=) and get some free credit to get you started.
```

Comment:
This wasn't a typo as credit can be valid both singular and plural

Reply:
Okay Sure Sir
I will try to contribute more